platform/vcs-impl/src/com/intellij/openapi/vcs/changes/committed/CommittedChangesPanel.java

```diff
@@ -102,10 +104,13 @@ public CommittedChangesPanel(Project project, final CommittedChangesProvider pro
     toolbarPanel.add(toolBar.getComponent());
     toolbarPanel.add(Box.createHorizontalGlue());
     myRegexCheckbox = new JCheckBox(VcsBundle.message("committed.changes.regex.title"));
+    // FIXME: take mnemonic from VcsBundle too ???
+    myRegexCheckbox.setMnemonic('g');		// same as in Editor > Find toolbar
```

Comment:
You can use the "&" symbol to specify the mnemonic char in the bundle, e.g. `Re&gexp`. I've modified your patch like that.

Btw, `setMnemonic(char)` is obsolete, the preferred method is `setMnemonic(int, e.g. VK_G)`